main.py

```diff
@@ -6,7 +6,8 @@
 
 
 def set_action_output(name: str, value: str):
-    print(f'::set-output name={name}::{value}')
+    with open("$GITHUB_OUTPUT", "a") as github_output_file:
```

Comment:
To get `$GITHUB_OUTPUT` env value, you should use `os.environ[key]` or `os.getenv(key, default)`.

```suggestion
    # See https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
    path = os.getenv('GITHUB_OUTPUT')
    if not path:
        print_action_error('$GITHUB_OUTPUT env is required.')
        exit(1)
    with open(path, 'a') as github_output_file:
```
I thinks this should be ok.

Reply:
OMG That makes so much sense! I found weird they called the output file as `$GITHUB_OUTPUT`...